wordle-web/src/views/IndexView.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-card>
+    <main>Dictionary</main>
+  </v-card>
+  <v-table>
+    <thead>
+      <v-text-field v-model="input" label="Label" @keyup="getInput()" variant="solo"></v-text-field>
```

Comment:
```suggestion
      <v-text-field v-model="input" label="Search" @keyup="getInput()" variant="solo"></v-text-field>
```

Reply:
This should do a live search as you type, rather than waiting for the user to hit 'enter'